app/assets/stylesheets/michigan-benefits/atoms/_typography.scss

```diff
@@ -1,6 +1,7 @@
 .text--section-header {
   color: $color-sky;
   font-weight: 600;
+  margin-bottom: 1em;
```

Comment:
this is necessary as we replace `<p>` tags with `<legend>` tags. `<p>` tags have a `1em` margin bottom, which should presumably be the same for any section header.